eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -622,10 +625,14 @@ logRatificationChecks gaId = do
     unlines
       [ "----- RATIFICATION CHECKS -----"
       , "prevActionAsExpected:\t" <> show (prevActionAsExpected gasAction ensPrevGovActionIds)
-      , "validCommitteeTerm:\t" <> show (validCommitteeTerm ensCommittee ensCurPParams currentEpoch)
+      , "validCommitteeTerm:\t" <> show (validCommitteeTerm gasAction ensCurPParams currentEpoch)
       , "notDelayed:\t\t??"
       , "withdrawalCanWithdraw:\t" <> show (withdrawalCanWithdraw gasAction curTreasury)
       , "committeeAccepted:\t" <> show (committeeAccepted ratEnv ratSt gas)
+      , "committeeAccepted:\t\t"
+          <> show (committeeAccepted ratEnv ratSt gas)
+          <> " [To Pass, at least: "
+          <> show (votingCommitteeThreshold ratSt gasAction)
```

Comment:
Just like with SPOs and DReps it will be useful to add the actual accepted ratio, not just the threshold:
```suggestion
          <> show (committeeAcceptedRatio ratEnv ratSt gas)
          <> " >= "
          <> show (votingCommitteeThreshold ratSt gasAction)
          <> "]"
```

Note, I am suggesting to change the type signatures (thus implementation) of all three functions to be consistent:

* `committeeAcceptedRatio :: RatifyEnv era -> RatifyState era -> GovActionState era -> Rational`
* `dRepAcceptedRatio :: RatifyEnv era -> GovActionState era -> Rational`
* and `spoAcceptedRatio :: RatifyEnv era -> GovActionState era -> Rational`

Reply:
Funnily, I'm pretty sure this is why I duplicated the line above, because I intended to do exactly this. But then something went wrong... Let me try again. Thanks!